addons/hr_recruitment_skills/static/src/components/search_job_applicant_menu/search_job_applicant_menu.js

```diff
@@ -0,0 +1,53 @@
+/** @odoo-module **/
+
+import { Component, markup } from "@odoo/owl";
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import { DropdownItem } from "@web/core/dropdown/dropdown_item";
+import { STATIC_ACTIONS_GROUP_NUMBER } from "@web/search/action_menus/action_menus";
+
+const cogMenuRegistry = registry.category("cogMenu");
+
+/**
+ * 'Search Matching Applicants' menu
+ *
+ * This component is used to search matching skills among the all applicants
+ * It's only available in the kanban and list view of the applicants.
+ * @extends Component
+ */
+export class SearchJobApplicant extends Component {
+    static template = "hr_recruitment_skills.SearchJobApplicant";
+    static components = { DropdownItem };
+    static props = {}
+
+    setup() {
+        this.action = useService("action");
+    }
+
+    //---------------------------------------------------------------------
+    // Protected
+    //---------------------------------------------------------------------
+
+    async openMatchingJobApplicants() {
+        const { globalContext } = this.env.searchModel;
+        const action = await this.env.services.orm.call(
+            'hr.job',
```

Comment:
In js and according to the odoo js linter, the convention is double quotes (same below in this file)
```suggestion
            "hr.job",
```